lib/parse/index.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * parse/index.js - JOSE Parser Entry Point
+ *
+ * Copyright (c) 2015 Cisco Systems, Inc. See LICENSE file.
+ */
+"use strict";
+
+var compact = require("./compact"),
+    json = require("./json");
+
+var parse = module.exports = function(input) {
+  if ("string" === typeof input) {
```

Comment:
All that said, are `Buffer`s used explicitly elsewhere in the Jose family? I've had the Buffer shim have weird side effects in browsers, so if we're not already doing something that guarantees it's in the bundle, maybe it's best to leave the extra convenience out.